File: CS/NavKB2/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web.Mvc;

namespace NavKB2.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            return View();
        }
        public ActionResult ShowFirst() {
            return View("SampleView1");
        }
        public ActionResult ShowSecond() {
            return View("SampleView2");
        }
    }
}

File: VB/NavKB2/Controllers/HomeController.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web.Mvc

Namespace NavKB2.Controllers
	Public Class HomeController
		Inherits Controller
		Public Function Index() As ActionResult
			Return View()
		End Function
		Public Function ShowFirst() As ActionResult
			Return View("SampleView1")
		End Function
		Public Function ShowSecond() As ActionResult
			Return View("SampleView2")
		End Function
	End Class
End Namespace